.bazelrc

```diff
@@ -1,2 +1,4 @@
+startup --host_jvm_args=-Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts \
+        --host_jvm_args=-Djavax.net.ssl.trustStorePassword=changeit
```

Comment:
I had to add these settings as a workaround for Bazel issues, but it should probably be removed before merge. Source: https://github.com/bazelbuild/bazel/issues/5741#issuecomment-418071387